inst/include/common/fims_math.hpp

```diff
@@ -86,6 +86,24 @@ inline const T logistic(const T &median, const T &slope, const T &x) {
   return (1.0) / (1.0 + exp(-1.0 * slope * (x - median)));
 }
 
+/**
+ * @brief The general double logistic function
+ *
+ * \f$ \frac{1.0}{ 1.0 + exp(-1.0 * slope_{1} (x - median_{1}))} \left(1-\frac{1.0}{ 1.0 + exp(-1.0 * slope_{2} (x - median_{2}))}  \right)\f$
+ *
+ * @param median1 the median (inflection point) of the ascending limb of the double double logistic function
```

Comment:
nit: I'm wondering if it would be more descriptive to name these args something like slope_asc and slope_desc instead of slope_1 and slope_2 so it's more clear to the user which is which (I get confused easily when numbering params).